recipes/eko/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "eko" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/N3PDF/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f7f6d7faed207bbbdbbbf003348e1afdcf823196ecea0156730f0097a3686ef3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py<38]
```

```suggestion
  noarch: python
```
Since this is pure python it seems like this can be noarch. If one of the dependencies is missing on Win that can be ignored right now.

---

I suspect it might be a problem with [`poetry`](https://python-poetry.org/), somehow it's failing to properly generate the entry points (scripts) in a format that is Windows-friendly. But I should check in more details.